Release notes for 4.20.1 (#17480)
docker/docs: desktop/release-notes.md

For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)

## 4.20.1

{% include release-date.html date="2023-06-05" %}

{% include desktop-install.md all=true version="4.20.1" build_path="/" %}

### Bug fixes and enhancements

#### For all platforms

> - Fix `docker load` of an image that contains attestations with the containerd integration.

Reviewer: ```suggestion
- contained image store: Fixed a bug that caused `docker load` to fail when loading an image that contains attestations.
```

> - Fix the default image exporter during build with the containerd integration.

Reviewer: ```suggestion
- containerd image store: Fixed the default image exporter during build.
```

#### For Windows

> - Fixed a bug when parsing the WSL version on the host in non-western locales. Fixes [docker/for-win#13518](https://github.com/docker/for-win/issues/13518) and [docker/for-win#13524](https://github.com/docker/for-win/issues/13524).

Reviewer: ```suggestion
- Fixed a bug that made it difficult to parse the WSL version on the host in non-western locales. Fixes [docker/for-win#13518](https://github.com/docker/for-win/issues/13518) and [docker/for-win#13524](https://github.com/docker/for-win/issues/13524).
```
